src/include/network/lan/address.rb

```diff
@@ -956,8 +910,9 @@ def wireless_tab
 
     # Dialog for setting up IP address
     # @return dialog result
-    def AddressDialog
-      initialize_address_settings
+    def AddressDialog(builder: nil)
```

Comment:
what is a goal of nil builder? does not it fail latter in builder.type?

Reply:
right, the parameter should be mandatory and defined

Reply:
replaced